tsconfig.base.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist/",
+    "sourceMap": true,
+    "strict": true,
+    "noImplicitReturns": true,
+    "noImplicitAny": true,
+    "module": "es6",
+    "moduleResolution": "node",
+    "target": "es2015",
+    "allowJs": true,
+    "jsx": "react",
+    "downlevelIteration": true,
+    "strictNullChecks": true,
+    "allowSyntheticDefaultImports": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true
+  },
+  "include": ["./src/**/*", "typings.d.ts", "../common/utils/data.utils.ts"]
```

Comment:
We should move the `include` fields back to the tsconfigs because they are project-specific